sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -31,6 +31,26 @@ public FormRecognizerClientLiveTests(bool isAsync) : base(isAsync)
         {
         }
 
+        /// <summary>
```

Comment:
Just to confirm - this moved in the file, but nothing else changed?

Reply:
Yes. This is just my preference and happy to undo the change.

Reply:
It's fine to keep it, I just wanted to make sure I wasn't missing another change.